budget app backend/controllers/obligatory_controller.js

```diff
@@ -0,0 +1,36 @@
+module.exports = class {
+    static obligatory_GET(req, res) {
```

Comment:
Use more meaningful method names!
Get by id  and get all - two different endpoints:
router.get("/:id", getById)
router.get("", getAll)

Please, change for all routes